Tests/CommandsCoreTests/CommandExecutorTests.swift

```diff
@@ -10,19 +10,34 @@ import XCTest
 
 class CommandExecutorTests: XCTestCase {
 
-    func testCommandExecutorExecutes() {
-        let commandExecutor = CommandExecutor(launchPath: "/bin/echo", arguments: ["hello world"])
-        XCTAssertNoThrow(commandExecutor.execute())
+    let commandExecutor = CommandExecutor()
+
+    func testCommandExecutorExecutesCommandWithArguments() {
+        let command = Command(launchPath: "/bin/echo", arguments: ["hello world"])
+        XCTAssertNoThrow(commandExecutor.execute(command))
     }
 
-    func testCommandExecutorLaunchpath() {
-        let commandExecutor = CommandExecutor(launchPath: "/bin/echo", arguments: [])
-        XCTAssertEqual(commandExecutor.launchPath, "/bin/echo")
+    func testCommandExecutorExecutesCommand() {
+        let command = Command(launchPath: "/bin/echo")
+        XCTAssertNoThrow(commandExecutor.execute(command))
```

Comment:
When would this `throw`? The [function signature](https://github.com/q231950/commands/pull/17/files#diff-5dde104951dd9021903c742058d07ed8R19) is not marked with `throws`.

Reply:
This is a good question. I'll have a look.